### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by romkatsu**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | Fix tests

### CODE REVIEW

File: config/packages/yiisoft/view/params.php [hunk 1]
```diff
@@ -1,20 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-use Yiisoft\Assets\AssetManager;
-use Yiisoft\Definitions\Reference;
-
-return [
-    'yiisoft/view' => [
-        'basePath' => '@views',
```
Comment (samdark) [inline config/packages/yiisoft/view/params.php:10]: That would not be set from the package defaults. Are you sure we don't need it?

File: config/packages/yiisoft/view/params.php [hunk 2]
```diff
@@ -1,20 +0,0 @@
...
-        'commonParameters' => [
-            'assetManager' => Reference::to(AssetManager::class),
```
Comment (vjik) [inline config/packages/yiisoft/view/params.php:12]: Seems `$assetManager` used in views.

**Review by samdark — CHANGES_REQUESTED** [review]
The config should not be deleted. It is setting an alias for Swagger.

File: config/packages/yiisoft/view/params.php (hunk 1 above)
Comment (romkatsu) [reply config/packages/yiisoft/view/params.php:10]: Not used

**Comment by romkatsu** [comment]
> The config should not be deleted. It is setting an alias for Swagger.

Fixed

**Pull request merged**